Emby.Server.Implementations/IO/ThrottledStream.cs

```diff
@@ -1,355 +0,0 @@
-using System;
```

Comment:
why this file is gone?

Reply:
1) Wasn't being used
2) Shouldn't be used as it's a big hack